tests/tools.py

```diff
@@ -168,6 +168,10 @@ def i_am_root():
     return os.geteuid() == 0
 
 
+def is_python_version_26():
```

Comment:
I needed this function for the tests I was writing and noticed it already existed in `test_crypt_util.py` so I refactored it into this common place for easier re-usability. `tools.py` seemed like a good place since it already contains other helper methods for our tests. Do you have a different idea in mind?

Reply:
Yes, create a separate module.

Reply:
Will send a separate PR for this.